src/node/hooks/express.ts

```diff
@@ -160,10 +165,10 @@ exports.restartServer = async () => {
   }
 
   // Measure response time
-  app.use((req, res, next) => {
+  app.use((req:any, res:any, next:Function) => {
```

Comment:
What if `req` doesn't have any specifique type ? And what if we type `req: Request`

[REF](https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.request.html)